### DISCUSSION TITLE
Folder template

### DISCUSSION THREAD

**Discussion by Rehner** [elrumo/macOS_Big_Sur_icons_replacements#1341] [General]
Hi,

 is there an folder template around? Would love to be able to create own folder icons.

Cheers Nicolas

**Comment by samdze** [comment]
<details>
<summary>Folder template</summary>

![Folder icon](https://user-images.githubusercontent.com/19392104/145657225-7c6dd537-5b22-48e0-94e6-3234d6396c4b.png)

</details>

Here you go, I use this one.
